doc/nrf/ug_radio_coex.rst

```diff
@@ -0,0 +1,111 @@
+.. _ug_radio_coex:
+
+Working with WiFi Coexistence in Short Range Radio applications
+###############################################################
+
+.. contents::
+   :local:
+   :depth: 4
+
+Short Range RF technologies such as BLE or 802.15.4 (from now on refered to as SR) use different radios than WiFi. However, if both SR and WiFi
+attempt to transmit at the same time RF waves will interfere with each other leading to decreased performance and uneeded higher power
+consumption. Also, in some cases (such as receiving ACK) no radio should be transmitting in order to ensure correct reception.
+
+In order to mitigate coexistence issues and improve coexistence performance a Packet Traffic Arbiter (PTA) is used. Both SR and WiFi request access to RF
+and PTA grants/denies that access.
+
+This guide describes how to add WiFi Coexistence (CX) support to your SR application in |NCS|.
+
+
+Software support
+****************
+
+If your application uses SR protocols and requires coexistence with WiFi, you must enable :ref:`nrfxlib:mpsl_cx` in the
+:ref:`nrfxlib:mpsl` (MPSL) library.
+
+CX with MPSL
+============
+
+The MPSL library provides an implementation for the WiFi Coexistens module for SR applications.
+To use this implementation, your application must use a protocol driver that enables the WiFi Coexistence (CX) feature.
+The library provides multiprotocol support, but you can also use it in applications that require only one protocol.
+To avoid conflicts, check the protocol documentation to see if the protocol uses CX support provided by MPSL.
+
+.. note::
+   Currently, the following protocols use CX support provided by MPSL:
+
+   * :ref:`ug_thread`
+   * :ref:`ug_zigbee`
+   * :ref:`ug_ble_controller`
+   * :ref:`ug_multiprotocol_support`
+
+.. _ug_radio_cx_requirements:
+
+Enabling MPSL
```

Comment:
This entire step should be done automatically by the build system. In my opinion there's no need to describe it here - an actual user doesn't need to perform any of the actions below

Reply:
Fine, we can leave for consistency and to clearly document that it's a prerequisite to enable WiFi Coex support.